css/popupStyle.css

```diff
@@ -0,0 +1,172 @@
+/* Project Popup */
+.popuptopContainer,
+.popupotherContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  display: none;
+  z-index: 2;
+}
```

Comment:
- [ ] Great job so far @akezeth and @maryam0007 :muscle: There are just a few style adjustments you need to make and your project will be complete :100: 
- [ ] I have pointed them out in the images below :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/232239587-4afd14ae-79f1-4080-8457-e56c4a45a2e9.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/232239668-904f45f7-1f58-4845-b10d-2e8fd318640b.png)

- [ ] Lastly, I noticed that when the page loads for the first time, the popup opens and close as expected. However, after opening a second popup, the close button doesn't work as expected so kindly fix this so users don't get frustrated when visiting your page :100: , I have attached a video reference below :100: 

[Screencast from 04-15-2023 05:06:42 PM.webm](https://user-images.githubusercontent.com/102699958/232241094-24643338-1f49-49c2-bdb7-da1e95364d34.webm)